project_event/models/project_task.py

```diff
@@ -120,9 +120,21 @@ def create(self, vals):
             if vals['activity_task_type'] == 'activity':
                 vals['code'] = self.env['ir.sequence'] \
                     .next_by_code('project.task.activity')
+                return_create = super(Task, self).create(vals)
+                vals['parent_id'] = return_create.id
+                vals['message_follower_ids'] = None
+                self.create_main_task(vals)
             elif vals['activity_task_type'] == 'task':
                 vals['code'] = self.env['ir.sequence'] \
                     .next_by_code('project.task.task')
+                return_create = super(Task, self).create(vals)
+        return return_create
+
+    @api.multi
+    def create_main_task(self, vals):
```

Comment:
@eilst the new created task is displayed in the activity list of the event

Reply:
done